tests/conftest.py

```diff
@@ -323,7 +323,7 @@ def fixture_parameters(
                 SubstanceParameter(
                     name=f"Substance_1_{encoding}",
                     data=mock_substances,
-                    encoding=encoding,
+                    encoding=encoding.name,
```

Comment:
is this encessary? if so, the conversion emchanismw as changed or the enum is not dieally designed
int he current version we can provide both the enum singleton and the string which will be autoconverted

Reply:
I fixed the conversion to work with current enum setup